domains/certificates/Query.API/API.AppTests/CertificateIssuingTests.cs

```diff
@@ -14,17 +15,25 @@
 
 namespace API.AppTests;
 
-public sealed class CertificateIssuingTests : IClassFixture<QueryApiWebApplicationFactory>, IClassFixture<MartenDbContainer>, IDisposable
+public sealed class CertificateIssuingTests :
+    IClassFixture<QueryApiWebApplicationFactory>,
+    IClassFixture<MartenDbContainer>,
+     IClassFixture<RabbitMqContainer>,
+    IDisposable
 {
     private readonly QueryApiWebApplicationFactory factory;
     private readonly DataSyncWireMock dataSyncWireMock;
 
-    public CertificateIssuingTests(QueryApiWebApplicationFactory factory, MartenDbContainer martenDbContainer)
+    public CertificateIssuingTests(QueryApiWebApplicationFactory factory, MartenDbContainer martenDbContainer, RabbitMqContainer rabbitMqContainer)
     {
-        dataSyncWireMock = new DataSyncWireMock(port: 9002);
+        dataSyncWireMock = new DataSyncWireMock(port: 9003);
```

Comment:
Side note: I remember now why I set this to 9002. ContractTests uses port 9001. Since it is possible that CertificateIssuingTests and ContractTests execute in parallel, then the port should unique for the test class.
I will try to see if there is a way to both randomize the port and check if the port is open in Wiremock

Reply:
No. But removing the port number inside DataSyncWireMock should mean that it gets a random port assigned. I have not tried if there should be any issues. You are welcome to do so. If you choose not, then close the conversation.

Reply:
I tried just removing the port, but our tests then fails. We/I have to do a bit more investigation to why that is. But that is outside of the scope of this task. I am closing the conversation.